src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -435,14 +437,14 @@ csm::EcefVector UsgsAstroFrameSensorModel::getSensorVelocity(const csm::ImageCoo
 
 csm::EcefVector UsgsAstroFrameSensorModel::getSensorVelocity(double time) const {
```

Comment:
If it's not used, is it still necessary to pass in `time`? Or is this part of the CSM api?

Reply:
This is part of the API